cirq-core/cirq/protocols/json_serialization.py

```diff
@@ -160,8 +160,7 @@ class name via a dot (.)
 
 
 # Copying the Python API, whose usage of `repr` annoys pylint.
-# TODO(#3388) Add documentation for Args.
-# pylint: disable=redefined-builtin, missing-param-doc
+# pylint: disable=redefine-builtin
```

Comment:
Should be: `redefined-builtin`

(Applies on l.222 below, too.)

Reply:
fixed